zio-http/shared/src/main/scala/zio/http/template2/Dom.scala

```diff
@@ -154,6 +154,22 @@ object Dom {
 
   def boolAttr(name: String, enabled: Boolean = true): BooleanAttribute = BooleanAttribute(name, enabled)
 
+  def multiAttr(name: String): PartialMultiAttribute =
+    PartialMultiAttribute(name, AttributeSeparator.Space)
```

Comment:
Reuse the other `multiAttr` method maybe?

```suggestion
    multiAttr(name, AttributeSeparator.Space)
```